source/photo.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Пинк</title>
+  </head>
+  <body>
+    <header class="page-header page-header--photo">
+      <div class="page-header__wrapper">
+        <a class="page-header__logo" href="#">
+          <img class="page-header__logo-image" src="" width="74" height="23" alt="На главную страницу сайта приложения Пинк.">
+        </a>
+        <nav class="page-header__navigation main-navigation">
+          <button class="main-navigation__toggle" type="button">
+            <span class="visually-hidden">Открыть меню.</span>
+          </button>
+          <ul class="main-navigation__list site-list">
+            <li class="site-list__item">
+              <a>Главная</a>
+            </li>
+            <li class="site-list__item">
+              <a href="#">Фотографии</a>
+            </li>
+            <li class="site-list__item">
+              <a href="#">Конкурс</a>
+            </li>
+            <li class="site-list__item">
+              <a href="https://htmlacademy.ru/intensive/adaptive" target="_blank">HTML Academy</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+
+      <h1 class="page-header__header page-header__header--photo">Фотографии пользователей</h1>
+    </header>
+
+    <main class="page-main">
+      <p class="page-main__description">
+        Взгляните на фотографии, которые выкладывают пользователи! Видите, как не хватает ваших?
+      </p>
+
+      <section class="photos">
+        <h2 class="visually-hidden">Лента фотографий</h2>
+        <ul class="photos__feed photos-list">
+          <li class="photos-list__item">
+            <a href="#"><img class="photos-list__picture photos-list__picture--top" width="320" height="137" src="" alt="Фотография Анны"></a>
```

Comment:
```suggestion
            <a href="#"><img class="photos-list__picture" width="320" height="137" src="" alt="Фотография Анны"></a>
```

топом будет лишка, а не картинка

Reply:
```suggestion
            <a href="#"><img class="photos-list__picture" width="320" height="137" src="" alt="Фотография Анны."></a>
```

поставь везде точки в альты